src/main/webapp/CreateProfilePage.html

```diff
@@ -36,12 +44,49 @@ <h1 class="text-center">Create Your Profile Page</h1>
             class="form-control"
             name="bio"
             maxlength="250"
+            required
           /><br />
+          <div class="invalid-feedback">
+            Please provide a bio.
+          </div>
         </div>
         <div class="text-center">
-          <input type="submit" class="btn btn-dark btn-lg" value="Submit" />
+          <input
+            type="submit"
+            class="btn btn-dark btn-lg"
+            value="Create Profile"
+          />
         </div>
       </form>
     </div>
+
+    <script>
```

Comment:
Same thing as the other comment, put the script in the JS file and wrap your code with jQuery ```$(document).ready()```